website/pages/blog/introducing-cloudquery-sdk.md

```diff
@@ -10,7 +10,7 @@ import { BlogHeader } from "../../components/BlogHeader"
 
 <BlogHeader/>
 
-Today we are pleased to announce the release of [CloudQuery SDK](https://github.com/cloudquery/cq-provider-sdk)!
+Today we are pleased to announce the release of [CloudQuery SDK](https://github.com/cloudquery/plugin-sdk)!
```

Comment:
I think we should either delete this blog and write a new one or mark it as deprecated...